python/hail/context.py

```diff
@@ -355,6 +355,36 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
                    sites_only=False, store_gq=False, pp_as_pl=False, skip_bad_ad=False, compress=False):
         """Import .vcf files as VariantDataset
 
+        **Notes**
```

Comment:
Could you please add one basic example?

Something like this:

**Examples**

Import data from a VCF file:

>>> vds = hc.import_vcf('output/example2.vcf.bgz')